src/Http/Response.php

```diff
@@ -2708,7 +2708,9 @@ protected function _sendFile($file, $range)
         }
 
         $bufferSize = 8192;
-        set_time_limit(0);
+        if (array_search('set_time_limit', explode(',', ini_get('disable_functions'))) === false) {
```

Comment:
Can't a string search be done for "set_time_limit" using `strpos` instead of using `explode`?  I don't see it yielding a false positive.

Reply:
Yes, this would also be possible, I change it later. `disable_functions` is a comma separated list....